src/python/pants/rules/core/test.py

```diff
@@ -53,14 +52,10 @@ def fast_test(console, addresses):
 
 @rule(TestResult, [HydratedTarget])
 def coordinator_of_tests(target):
```

Comment:
This one doesn't strictly need to exist, no. I didn't inline it into `fast_test` here because I thought that it would distract from the rest of the PR.

Reply:
Let's do this either now or as a fast-follow :)